cookbook/service_container/shared.rst

```diff
@@ -0,0 +1,43 @@
+.. index::
+    single: Service Container; Shared Services
+
+How to Define Not Shared Services
```

Comment:
"not shared", "non shared", "unshared", ...? (in other words: I'm not sure how to call this)